source/frontend/src/features/properties/map/acquisition/detail/AcquistionOwnersSummaryContainer.tsx

```diff
@@ -0,0 +1,106 @@
+import { Api_AcquisitionFile, Api_AcquisitionFileOwner } from 'models/api/AcquisitionFile';
+import { Api_Address } from 'models/api/Address';
+import React from 'react';
+import { useCallback, useEffect, useState } from 'react';
+
+import { useAcquisitionProvider } from '../hooks/useAcquisitionProvider';
+import { DetailAcquistionFileOwner } from './models';
+
+export interface IAcquisitionOwnersContainerProps {
+  acquisitionFile?: Api_AcquisitionFile;
+  View: React.FC<IAcquisitionOwnersSummaryViewProps>;
+}
+
+export interface IAcquisitionOwnersSummaryViewProps {
+  ownersList?: DetailAcquistionFileOwner[];
+  isLoading: boolean;
+}
+
+const AcquisitionOwnersSummaryContainer: React.FunctionComponent<
+  React.PropsWithChildren<IAcquisitionOwnersContainerProps>
+> = ({ acquisitionFile, View }) => {
+  const [ownersDetails, setOwnersDetails] = useState<DetailAcquistionFileOwner[] | undefined>(
+    undefined,
+  );
+
+  const {
+    getAcquisitionOwners: {
+      execute: retrieveAcquisitionFileOwners,
+      loading: loadingAcquisitionFileOwners,
+    },
+  } = useAcquisitionProvider();
+
+  const fetchOwnersApi = useCallback(async () => {
+    let acquisitionOwners = await retrieveAcquisitionFileOwners(acquisitionFile?.id!);
+    let ownerDetailList = acquisitionOwners?.map(o => ({
+      ownerName: getOwnerDisplayName(o),
+      ownerOtherName: o.lastNameOrCorp2?.trim(),
+      ownerDisplayAddress: getFormatedAddress(o.address),
+    }));
+
+    setOwnersDetails(ownerDetailList);
+  }, [acquisitionFile, retrieveAcquisitionFileOwners]);
+
+  useEffect(() => {
+    if (ownersDetails === undefined) {
+      fetchOwnersApi();
+    }
+  }, [ownersDetails, fetchOwnersApi]);
+
+  return <View ownersList={ownersDetails} isLoading={loadingAcquisitionFileOwners} />;
+};
+
+export default AcquisitionOwnersSummaryContainer;
+
+export const getOwnerDisplayName = (owner: Api_AcquisitionFileOwner): string => {
```

Comment:
this looks duplicated from above, could move to an extracted shared function.

Reply:
Removed